src/dmd/dmsc.d

```diff
@@ -97,7 +98,7 @@ void backend_init()
         params.symdebug,
         dmdParams.alwaysframe,
         params.stackstomp,
-        params.cpu >= CPU.avx2 ? 2 : params.cpu >= CPU.avx ? 1 : 0,
+        target.cpu >= CPU.avx2 ? 2 : target.cpu >= CPU.avx ? 1 : 0,
```

Comment:
Note for future, could be encapsulated into a target hook, which would allow CPU to be made private to the target module (there would have to be target-specific command-line handling as well)

Reply:
>private to the target module

can't be. needs to be set by mars.d